lib/src/pagination/pagination.component.html

```diff
@@ -0,0 +1,99 @@
+                <li class="page-item" (click)="handleOnChange(value + 1)">
+                    <button class="page-link" title="nextText">
+                        <span class="nav-action">
+                            <ng-container *ngIf="useTextNav; else angleRightIcon"> {{ nextText ? nextText : "Next" }} </ng-container>
+                            <ng-template #angleRightIcon>
+                                <svg name="angle-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512">
+                                    <path
+                                        d="M166.9 264.5l-117.8 116c-4.7 4.7-12.3 4.7-17 0l-7.1-7.1c-4.7-4.7-4.7-12.3 0-17L127.3 256 25.1 155.6c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0l117.8 116c4.6 4.7 4.6 12.3-.1 17z"
+                                    />
+                                </svg>
+                            </ng-template>
+                        </span>
+
+                        <span class="sr-only">{{ nextText || "Next" }}</span>
+                    </button>
+                </li>
+            </ng-container>
+            <ng-container *ngIf="value !== pagingSize && useFirstAndLast">
+                <li class="page-item" (click)="handleOnChange(pagingSize)">
+                    <button class="page-link" title="lastText">
+                        <span class="nav-action">
+                            <ng-container *ngIf="useTextNav; else angleDoubleRightIcon">
+                                {{ lastText ? lastText : "Last" }}
+                            </ng-container>
+                            <ng-template #angleDoubleRightIcon class="na-action">
+                                <svg name="angle-double-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
+                                    <path
+                                        d="M166.9 264.5l-117.8 116c-4.7 4.7-12.3 4.7-17 0l-7.1-7.1c-4.7-4.7-4.7-12.3 0-17L127.3 256 25.1 155.6c-4.7-4.7-4.7-12.3 0-17l7.1-7.1c4.7-4.7 12.3-4.7 17 0l117.8 116c4.6 4.7 4.6 12.3-.1 17zm128-17l-117.8-116c-4.7-4.7-12.3-4.7-17 0l-7.1 7.1c-4.7 4.7-4.7 12.3 0 17L255.3 256 153.1 356.4c-4.7 4.7-4.7 12.3 0 17l7.1 7.1c4.7 4.7 12.3 4.7 17 0l117.8-116c4.6-4.7 4.6-12.3-.1-17z"
+                                    />
+                                </svg>
+                            </ng-template>
+                        </span>
+                        <span class="sr-only">{{ lastText || "Last" }}</span>
+                    </button>
+                </li>
+            </ng-container>
+        </ul>
+        <ng-container *ngIf="useDotNav">
+            <ul class="pagination dotnav">
+                <ng-container *ngFor="let num; dotnavList">
+                    <li class="page-item" [ngClass]="{ active: value === num }" (click)="handleOnChange(num)" value="num">
+                        <span class="page-dot-link" [ngClass]="{ active: value === num }"></span>
+                    </li>
```

Comment:
Why are you setting class `active` in parent and child? The parent is enough. 

Don't forget the binding
```suggestion
                    <li class="page-item" [class.active]="value === num" (click)="handleOnChange(num)" [value]="num">
                        <span class="page-dot-link"></span>
                    </li>
```